app-editors/featherpad/featherpad-1.3.4.ebuild

```diff
@@ -16,9 +16,9 @@ KEYWORDS="~amd64 ~riscv ~x86"
 IUSE="+X"
 
 RDEPEND="app-text/hunspell:=
-	>=dev-qt/qtcore-5.15.0
+	>=dev-qt/qtcore-5.15.0:5
+	dev-qt/qtdbus:5
 	dev-qt/qtgui:5
-	dev-qt/qtnetwork:5
 	dev-qt/qtprintsupport:5
 	dev-qt/qtsvg:5
 	dev-qt/qtwidgets:5
```

Comment:
If you are looking at most other Qt revdeps, you'll notice they have uniform minimum version req across all of their `dev.qt/*` dependencies. It is not blocking this PR from being merged though.

Reply:
So they just repeat the version req. on all the dependencies? Seems like unnecessary duplication, but I will of course fix this for the next release. (It has been like this since 1.3.1.) Thx for the review, by the way.

Reply:
It is to ease Portage's dependency calculation/interpretation of potential conflict output, since the main module of Qt is split into several packages in Gentoo. But also, packages never depend on just one part of Qt with a minimal version but all of them, so not enforcing it on the others is just incomplete.

Basically, think of the user that has `app-editors/featherpad` as their *only* Qt-based package installed - an emerge featherpad will not reliably require all necessary minimum deps.